pkg/handler/handler.go

```diff
@@ -335,6 +337,37 @@ func (h handler) apbAddSpec(w http.ResponseWriter, r *http.Request, params map[s
 	writeDefaultResponse(w, http.StatusOK, resp, err)
 }
 
+func (h handler) apbRemoveSpec(w http.ResponseWriter, r *http.Request, params map[string]string) {
+	ansibleBroker, ok := h.broker.(broker.DevBroker)
+	if !ok {
+		h.log.Errorf("unable to use broker - %T as ansible service broker", h.broker)
```

Comment:
I don't think `Errorf` is actually required. I originally thought I had to `fmt.Sprintf`, and realized they just accept format strings. `Error("thing %s", "fubar")`, unless I'm off my rocker.

We should just try to remain consistent.